daemon/graphdriver/driver_linux.go

```diff
@@ -118,6 +118,9 @@ func (c *defaultChecker) IsMounted(path string) bool {
 func Mounted(fsType FsMagic, mountPath string) (bool, error) {
 	var buf unix.Statfs_t
 	if err := unix.Statfs(mountPath, &buf); err != nil {
+		if err == unix.ENOENT { // not exist, thus not mounted
```

Comment:
does `os.IsNotExist(err)` work for this?

Reply:
Yes, they are both equally good, I like comparison of two integers slightly better than a function call. `os.IsNotExists` is wrapper that's also suitable for os.* functions returning `PathError`, `LinkError` etc. In this case it's a pure system call which returns an integer (retyped as `unix.Errno`).